AppCenter/AppCenter/Internals/Channel/MSChannelGroupDefault.m

```diff
@@ -27,7 +27,7 @@ - (instancetype)initWithIngestion:(nullable MSAppCenterIngestion *)ingestion {
     _storage = [MSLogDBStorage new];
     if (ingestion) {
       _ingestion = ingestion;
-      _ingestion.authToken = [MSAuthTokenContext sharedInstance].authToken;
+      _ingestion.authToken = [[MSAuthTokenContext sharedInstance] getAuthToken];
```

Comment:
The "getter" for the token should be named just `authToken` as per ObjC convention.